### PR TITLE
[1393][IMP] account_analytic_profit_loss_analysis: pre-commit, README

### PR DESCRIPTION
**Pull request by Ryoko04**
Task [1393](https://www.quartile.co/web#id=1393&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: account_analytic_profit_loss_analysis/models/account_analytic_line.py [hunk 1]
```diff
@@ -6,94 +6,86 @@
...
     )
     pj_id = fields.Many2one(
-        'project.project',
-        string='Related Project',
-        compute='_compute_pj_id',
-        inverse='_set_account_id',
+        "project.project",
+        string="Related Project",
+        compute="_compute_pj_id",
+        _inverse_="_set_account_id",
```
Comment (yostashiro) [inline account_analytic_profit_loss_analysis/models/account_analytic_line.py]: @Ryoko04 Can you please check if this method is triggered as expected? Haven't seen `_inverse_` thing before.

@TimLai125 Is this a valid change?

Comment (Ryoko04) [reply account_analytic_profit_loss_analysis/models/account_analytic_line.py]: @yostashiro 
Yes, I should have changed  to 'inverse="_inverse_set_account_id"'. I noticed it now.
image

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Review by TimLai125 — APPROVED** [review]
LGTM

**Pull request merged**